helpers/talk_helpers.rb

```diff
@@ -0,0 +1,83 @@
+module TalkHelpers
+  def render_talks(for_page = current_page)
+    year = current_page.data.meeting_date.year.to_s
+    month = current_page.data.meeting_date.strftime('%B').downcase
+    talks = find_talks(year, month)
+    talks&.map { it.render(on: self) }.join
```

Comment:
I don't think talks would ever be nil, and if it did you would end up doing nil.join. I'd drop the `&.`

Reply:
There is a `nil` problem, but the `&.` isn't in a place where it'd be useful.  The `dig` in `find_talks` can return `nil` so the `.map` there would explode, not the `.map` here.  I'll rejig this to be more robust and return a "No talks yet" message.